test/assert-es.js

```diff
@@ -0,0 +1,15 @@
+export default {
+for: Class => {
+    console.log('%c' + Class, 'font-weight:bold');
+    return (expression, expected, message = '') => {
+      if (!Object.is(expression, expected)) {
+        if (message)
+          console.error('', message);
+        console.error('', 'Expected:', String(expected));
+        console.error('', 'Got instead:', String(expression));
+        console.error('');
+        process.exit(1);
```

Comment:
nope, I think this still need to go away from here

Reply:
Actually ... you know what? I'll drop this, as I need to go soon, no need to hang in here

Reply:
Ah yeah I missed that, my bad.